shared_requirements.txt

```diff
@@ -77,6 +77,7 @@ azure-mgmt-subscription~=0.2.0
 azure-mgmt-trafficmanager~=0.50.0
 azure-mgmt-web~=0.35.0
 azure-nspkg
```

Comment:
Do you need to remove this?

Reply:
I don't believe so. `azure-nspkg` is for the `azure` namespace. I'm creating a new package for `azure.keyvault`.